G/glm/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "glm"
+version = v"9.9.9"
```

Comment:
Looks like the version number is [0.9.9.8](https://github.com/g-truc/glm/releases/tag/0.9.9.8), not really 9.9.9.  However we have to stick to the format `X.Y.Z`, so maybe 0.9.9?

Reply:
there hasn't been a release yet for 0.9.9.9, but i'm using it anyway from the master branch because the 0.9.9.8 tarball's CMakeLists.txt doesn't have an install target.

i think 0.9.9 (essentially omitting the 4th digit) is more future-proof than 9.9.9, so i'll go ahead and make that change.